sbin/common/config_init.sh

```diff
@@ -82,7 +82,10 @@ NUM_PROCESSORS
 OPENJDK_BUILD_NUMBER
 OPENJDK_CORE_VERSION
 OPENJDK_FEATURE_NUMBER
+OPENJDK_FOREST_ALTID
```

Comment:
I think you mean `--version` ? Thats what I will use instead of  OPENJDK_FOREST_ALTID. The directory/tarball should remain the only mandatory argument, which was name of the repo untill now.